packages/react-pose/rollup.config.js

```diff
@@ -59,21 +59,26 @@ const umdProd = Object.assign({}, umd, {
 });
 
 const es = Object.assign({}, config, {
+  experimentalCodeSplitting: true,
   output: {
-    file: 'dist/react-pose.es.js',
+    dir: 'es',
     format: 'es',
-    exports: 'named'
+    exports: 'named',
+    entryNames: '[alias].js',
   },
-  plugins: [typescript(noDeclarationConfig)]
+  plugins: [typescript(typescriptConfig)]
 });
 
 const cjs = Object.assign({}, config, {
+  experimentalCodeSplitting: true,
   output: {
-    file: 'lib/index.js',
+    dir: 'lib',
     format: 'cjs',
-    exports: 'named'
+    exports: 'named',
+    entryNames: '[alias].js'
   },
+  // plugins: [typescript(declarationsConfig)]
```

Comment:
unfortunately both this config and umd builds are failing due to some typing issues, could u check them out? I'm not well versed in TS and I have no idea why they are failing